app/controllers/organisation_relationships_controller.rb

```diff
@@ -93,24 +93,37 @@ def create_managing_agent
     redirect_to managing_agents_organisation_path(related_organisation_id:)
   end
 
+  def remove_housing_provider
+    render "organisation_relationships/remove_housing_provider", layout: "application"
```

Comment:
```suggestion
```

You shouldn't need to be explicit about the render here. All of those are implicit as the action name matches the template name.